pkg/kstatus/polling/statusreaders/common.go

```diff
@@ -173,10 +173,10 @@ func toSelector(resource *unstructured.Unstructured, path ...string) (labels.Sel
 	return metav1.LabelSelectorAsSelector(&s)
 }
 
+// toIdentifier extracts to identifying information from an unstructured object
```

Comment:
Maybe we can rid of this `toIdentifier` function since it does not add anything to the regular `UnstructuredToObjMeta`.